pkgs/applications/video/kodi/build-kodi-binary-addon.nix

```diff
@@ -28,7 +29,10 @@ toKodiAddon (stdenv.mkDerivation ({
     runHook preInstall
 
     make install
-    ln -s $out/lib/addons/${n}/${n}.so.${version} $out${addonDir}/${n}/${n}.so.${version}
+
+    test -f $out/lib/addons/${n}/${n}.so && ln -s $out/lib/addons/${n}/${n}.so $out${addonDir}/${n}/${n}.so || true
+    test -f $out/lib/addons/${n}/${n}.so.${version} && ln -s $out/lib/addons/${n}/${n}.so.${version} $out${addonDir}/${n}/${n}.so.${version} || true
```

Comment:
I would prefer the [[ ... ]] syntax here.

Reply:
Changed as requested.